### ISSUE TITLE
Dark Gem fabrication

### ISSUE THREAD

**Issue by belathus** [belathus/Wanderlust-Renewed#204]
Obviously, dark gems are a part of ore generation.

One who gets deeply into EvilCraft will need a ton of dark gems. If we close issue #203 as currently suggested, there will likely be no renewable way to get dark gems (though you'll be able to get huge amounts of dark gems via an asteroid mining mission, excavator, or some sort of quarry). In my mind, this isn't an issue that needs resolving, but I do want to consider it.

In short, should we make it possible to fabricate dark gems? If so, how?

**Comment by Divineaspect** [comment]
Shadow gem in tier 2 blood infuser into dark gem block?

**Comment by belathus** [comment]
That would make them renewable. I'm good with that.

**Comment by Divineaspect** [comment]
It would also provide annother reason to go to or set up near Darkland Biomes

**Comment by belathus** [comment]
It looks like I also made it so you can duplicate dark gems using the conjuration catalyst for 2500 mana.
